### DISCUSSION TITLE
大家的梦想

### DISCUSSION THREAD

**Discussion by leslie158** [Benefitspace/dream#13] [Show and tell]
官网：https://benefitspace.github.io/dream/

我有一个梦想 希望持有dream的每个人都能变得富有，dreamcoin，静态收益模式，通过持有dream币获得源源不断的被动收益，只要市场上有交易你就能增加币的数量，自动复利，拿的越久赚的越多。依托强大的bsc公链，交易不断增加资金池，从而避免了价格归零的风险。交易燃烧10%筹码，5%分给所有持有人，5%加入资金池，你拿出的一小份，让生态持续盘活。
50%的流动性LP已经打入黑洞地址https://bscscan.com/tx/0xa6c7405e402e79f9d8c77bd3f4ba22f883ba678c6b5e967f804c1cfd9e6d270a 
购买地址：https://exchange.pancakeswap.finance/#/swap?outputCurrency=0x3bfE9F4e25ed2793610BD78aFe766871F62AA11c
电报群：https://t.me/dream_BSC
discord服务器：https://discord.gg/s4TzMfMRsf
价格查询：https://poocoin.app/tokens/0x3bfe9f4e25ed2793610bd78afe766871f62aa11c

**Comment by sean-go-ai** [comment]
去中心化加油
